identity/src/main/java/me/uport/sdk/identity/Account.kt

```diff
@@ -56,6 +56,23 @@ data class Account(
 
     fun getSigner(context: Context): Signer = UportHDSignerImpl(context, UportHDSigner(), rootAddress = handle, deviceAddress = deviceAddress)
 
+    /**
+     * This function generates the DID associated with an account based on the account type.
+     *
+     * Limitation: The current implementation only covers KeyPair and MetaIdentity Account types
+     * @return the DID as a string
+     * @throws IllegalStateException if there is no implementation for the Account Type
+     */
+    fun getDID(): String {
+        if (type.equals(AccountType.MetaIdentityManager)) {
```

Comment:
The logic is good but a more idiomatic approach is `when(type)` instead.

Even more, you can do `return when`

Reply:
Thanks